src/app/common/product-card/product-card.component.html

```diff
@@ -0,0 +1,33 @@
+<!--<div class="card" style="width: 18rem">
+  <img src="{{ movie.image }}" class="card-img-top" alt="Picture not found" />
+  <div class="card-body">
+    <h5 class="card-title">{{ movie.name }}</h5>
+    <p class="card-text">
+      {{ movie.description }}
+    </p>
+    <div class="data">
+      <h6>Stock</h6>
+      <p>{{ movie.stock }}</p>
+      <h6>Price</h6>
+      <p>{{ movie.price }}</p>
+    </div>
+  </div>
+</div>-->
+
+<div class="col mb-4">
+  <div class="card h-100">
+  <img src="{{ movie.image }}" class="card-img-top" alt="Picture not found" />
```

Comment:
TODO: Generate better alt text later, maybe alt="{{ movie.name }} poster"
Also note that there's a better way to set an HTML property in Angular with []